tracker/gioui/buttons.go

```diff
@@ -44,6 +44,14 @@ type (
 		TipArea   component.TipArea
 		Bool      tracker.Bool
 	}
+
+	MenuClickable struct {
+		Clickable Clickable
+		menu      Menu
+		Selected  tracker.OptionalInt
+		TipArea   component.TipArea
+		Tooltip   component.Tooltip
+	}
```

Comment:
the new "when putting MIDI into tracks, you can, optionally, select another track where the velocity byte goes" feature required a new type of button